etcd/etcd-live-http-to-https-migration.md

```diff
@@ -0,0 +1,196 @@
+# etcd Live HTTP to HTTPS migration
+
+If you already have existing etcd cluster which uses insecure HTTP connections and you would like to switch from HTTP to HTTPS and in addition enable TLS certificate authentication, you have to follow instruction below.
+
+## Prepare Cluster Components
+
+### Check insecure port availability
+
+By default fleet and flannel use two etcd client ports: 2379 (IANA port assignment) and 4001 (legacy port). Last one is used for etcd v0.4 backward compatibility. In this example we will use 4001 TCP port for insecure HTTP connections and our etcd members will listen that port only on local interface for security reason. This will allow us to switch transparently to HTTPS configuration without services downtime and simplify configuration
```

Comment:
This wasn't clear to me on the first read that this was the strategy.

@joshix I know this is throwing you into the fire, but do you have options on this?